core/src/main/java/org/ldaptive/auth/ext/ActiveDirectoryAuthenticationResponseHandler.java

```diff
@@ -115,9 +117,6 @@ public Period getExpirationPeriod()
    */
   public void setExpirationPeriod(final Period period)
   {
-    if (period == null) {
-      throw new IllegalArgumentException("Expiration cannot be null");
-    }
```

Comment:
The check is an artifact of the conversion to ```Period```, when negatives values were not allowed.
It's no longer correct, these variables can be null.

I foresee a use case where configuration is using the full constructor and allowing nulls to pass through will make for easier wiring.